### ISSUE TITLE
SpringSocialSecuritySignInDetails is not Serializable

### ISSUE THREAD

**Issue by michaellavelle** [socialsignin/spring-social-security#6]

**Comment by BinaryTweedDeej** [comment]
I'm experiencing this issue when try to persist sessions via memcached-session-manager. Looking at SpringSocialSecuritySignInDetails, ConnectionData implements Serializable, so there's no reason why SSSSIDetails itself can't - the only other instance variable is a String?

**Comment by michaellavelle** [comment]
Thanks for your feedback BinaryTweedDeej - it's prompted me to fix this issue and release 1.0.3.RELEASE as a patch.

My intention was to fix this issue as part of the recent refactoring - SpringSocialSecuritySignInDetails used to reference a Connection which wasn't Serialiable - I recently changed this to be ConnectionData which is Serializable, but for some reason the addition of the Serializable tag was left out of the last release.

1.0.3.RELEASE is now released with this fix - I hope this resolves your issue.

Cheers,,

Michael
